ray-operator/config/rbac/role.yaml

```diff
@@ -1,7 +1,9 @@
+
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  creationTimestamp: null
```

Comment:
why is this needed ?

Reply:
This must have been auto generated I will remove :+1: